daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-deadletter.md

```diff
@@ -0,0 +1,78 @@
+---
+type: docs
+title: "Dead Letter Topics"
+linkTitle: "Dead Letter Topics"
+weight: 2250
+description: "Use dead letter topics to capture undeliverable messages"
+---
+
+## Introduction
+
+Dead letter topics are used to hold messages that cannot be delivered to any receiver. This eases the pressure on receivers by freeing them from dealing with poison messages, while allowing developers to write code that reads from the dead letter topic and either fix the message or abandon it completely.
+
+When a dead letter topic is set, any message that failed to be delivered to an app for a configured topic will be put on the dead letter topic.
+
+Dapr enables dead letter topics for all of it's pub/sub components, even if the underlying system does not support this feature natively.
+
+## Configuring a dead letter topic through YAML
+
+The following YAML shows how to configure a dead letter topic named `poison` for messages consumed from the `orders` topic.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Subscription
+metadata:
+  name: orders
+spec:
+  topic: orders
+  route: /checkout
+  pubsubname: nats
```

Comment:
```suggestion
  pubsubname: order_pub_sub
```

Reply:
Changed to `pubsub` as `order-pub-sub` implies there's a single pub/sub for the `order` topic which usually isn't the case. (pub/subs are used for multiple topics).